packages/gatsby-plugin-sass/src/__tests__/__snapshots__/gatsby-node.js.snap

```diff
@@ -12,29 +12,20 @@ exports[`gatsby-plugin-sass Stage: build-html / No options 1`] = `
                 Object {
                   "test": /\\\\\\.module\\\\\\.s\\(a\\|c\\)ss\\$/,
                   "use": Array [
-                    "miniCssExtract",
```

Comment:
This seems kinda concerning.. ? I'm not 100% sure these snapshots are completely useful, but it seems like they're catching something valuable here!

Reply:
it is correct - this is for `build-html` stage where we don't extract css